aten/src/ATen/native/Sorting.cpp

```diff
@@ -188,9 +343,9 @@ void quantile_impl(
 
   Tensor ranks;
   if (ignore_nan) {
-    // For nanquantile, compute ranks based on number of non-nan values since sort
-    // compares nans as largest value. If all values are nan, set rank to 0 so that
-    // the quantile computed will be nan.
+    // For nanquantile, compute ranks based on number of non-nan values since
+    // sort compares nans as largest value. If all values are nan, set rank to 0
```

Comment:
nit: "treats NaNs as larger than any non-NaN values. If all values are NaN, sets rank to zero so the computed quantiles will be NaN, too."

One trick when writing comments is to ensure they describe what the code does ("sets the value to zero") as opposed to giving an order to the reader ("set the value to zero").